initializr-service/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM openjdk:8-jre-alpine
+MAINTAINER yaweiw@microsoft.com
```

Comment:
remove yaweiw@microsoft.com for now.
in the future, we'll create a azurespringcloud@microsoft.com DL which includes all of us.

Reply:
copy that.